forch/forch_metrics.py

```diff
@@ -80,6 +80,10 @@ def _add_vars(self):
                       'No. of RADIUS query responses received from server', Counter)
         self._add_var('process_state', 'Current process state', Gauge, labels=['process'])
 
+        learned_mac_help_text = 'MAC address stored as 64bit number to switch anme, port, VLAN, '\
```

Comment:
don't use line continuation, this string seems long to have here -- where does it show up?

Reply:
This is the HELP message that will be shown when scraping the metrics. Updated the message to better describe the new format.